CI failure in integration tests due to a traceback:

```
Traceback (most recent call last):
  File "/tmp/ansible_sYS9Fq/ansible_module_get_url.py", line 572, in <module>
    main()
  File "/tmp/ansible_sYS9Fq/ansible_module_get_url.py", line 546, in main
    os.remove(tmpsrc)
OSError: [Errno 2] No such file or directory: '/tmp/tmpGJT49S'
```

--- a/lib/ansible/modules/net_tools/basics/get_url.py
+++ b/lib/ansible/modules/net_tools/basics/get_url.py
@@ -526,9 +526,10 @@ def main():
             if backup:
                 if os.path.exists(dest):
                     backup_file = module.backup_local(dest)
-            shutil.copyfile(tmpsrc, dest)
+            module.atomic_move(tmpsrc, dest)
         except Exception as e:
-            os.remove(tmpsrc)
+            if os.path.exists(tmpsrc):
+                os.remove(tmpsrc)
             module.fail_json(msg="failed to copy %s to %s: %s" % (tmpsrc, dest, to_native(e)),
                              exception=traceback.format_exc())
         changed = True
@@ -541,8 +542,6 @@ def main():
         if checksum != destination_checksum:
             os.remove(dest)
             module.fail_json(msg="The checksum for %s did not match %s; it was %s." % (dest, checksum, destination_checksum))
-
-    os.remove(tmpsrc)
 
     # allow file attribute changes
     module.params['path'] = dest

